build-docker-images.sh

```diff
@@ -8,18 +8,30 @@ then
   exit 0
 fi
 
+# Build or fetch cached joystream/node docker image
+if [[ "$SKIP_JOYSTREAM_NODE" = 1 || "$SKIP_JOYSTREAM_NODE" = "true" ]]; then
+  echo "Skipping build of joystream/node docker image."
+else
+  # Fetch a cached joystream/node image if one is found matching code shasum instead of building
+  CODE_HASH=`scripts/runtime-code-shasum.sh`
+  IMAGE=joystream/node:${CODE_HASH}
```

Comment:
I take it we use the same `scripts/runtime-code-shasum.sh` to tag the image once it's created?

Reply:
correct that is don in our github actions here: https://github.com/Joystream/joystream/blob/master/.github/workflows/joystream-node-docker.yml#L18

Reply:
:+1: